kotlin/display-content-of-utility-network-container/src/main/java/com/esri/arcgisruntime/sample/displaycontentofutilitynetworkcontainer/MainActivity.kt

```diff
@@ -149,16 +153,20 @@ class MainActivity : AppCompatActivity() {
         // handle when map is clicked by retrieving the point
         mapView.onTouchListener =
             object : DefaultMapViewOnTouchListener(this@MainActivity, mapView) {
-                override fun onSingleTapConfirmed(motionEvent: MotionEvent?): Boolean {
-                    motionEvent?.let { event ->
-                        // create a point from where the user clicked
-                        android.graphics.Point(event.x.toInt(), event.y.toInt())
-                            .let { screenPoint ->
-                                // identify and handle the feature of the clicked at point
-                                handleMapViewClicked(screenPoint)
-                            }
+                override fun onSingleTapConfirmed(event: MotionEvent?): Boolean {
+                    if (event != null) {
```

Comment:
Just FYI related to https://github.com/Esri/arcgis-runtime-samples-android/pull/981#discussion_r876885733: I was referring more to the `android.graphics.Point(event.x.toInt(), event.y.toInt().let {` part. I think the ` motionEvent?.let { event ->` part is fine because it also does a null check and that's a common pattern. I just wasn't such a fan of using `let` a lot if it isn't an improvement over making a local variable. I'm not asking for further changes though, the code as you have it now is good :+1: